.swiftlint.yml

```diff
@@ -0,0 +1,11 @@
+# Set max line length before warning (default is 120)
+line_length: 240
+
+disabled_rules:
+  - trailing_whitespace # Disables SwiftLint complaining about whitespace characters on empty lines
+  - todo # Disables auto-warning of TODO statements
+
+identifier_name:
+  excluded:
+    - id
+    - ok
```

Comment:
Might be able to get rid of these exclusions.

Reply:
Good point.